scripts/keyword_paper_pickle_to_json.py

```diff
@@ -0,0 +1,12 @@
+import pickle
```

Comment:
What is this file used for?

Reply:
This file was just so I could convert my intermediate saved files into JSON and show the schema during our weekly presentations. I can remove it, because it isn't used for the final product.